config/locales/about/en.yml

```diff
@@ -11,15 +11,23 @@ en:
           or deciding to improve the way an area is managed to maximise benefits for both human well-being 
           and the species that live there." 
     
-    footer:
-      left: "In 2018, the governments of Egypt and China, with support from CBD Secretariat, launched the 
+    footer: In 2018, the governments of Egypt and China, with support from the CBD Secretariat, launched the 
             Sharm El-Sheikh to Kunming Action Agenda for Nature and People. Through the Action Agenda, 
             Governments agreed to encourage non-state actors to develop biodiversity commitments that would 
             contribute towards putting biodiversity on a path to recovery by 2030. The platform will 
-            continue to raise public awareness and inspire others to safeguard, restore and invest in nature."
-      
-      right: "The Commitments for Nature platform will complement and support the Action Agenda by working 
-              with it in tandem, and displaying the progress being made towards different commitments. 
-              With the help of the platform, non-state actors can exchange lessons learned, celebrate success 
-              and inspire each other. Commitments for Nature will provide an opportunity to facilitate 
-              partnerships and collaboration, as well as test innovative approaches to conservation."
+            continue to raise public awareness and inspire others to safeguard, restore and invest in nature.
+            The Commitments for Nature platform will complement and support the 
+            <a href='https://www.cbd.int/action-agenda/contribute/' target='blank'>Action Agenda</a> 
+            by working with it in tandem, and displaying the progress being made towards different commitments.
+            Another important international body that the Commitments for Nature Platform links to is the 
+            <a href='https://citieswithnature.org/how-do-cities-contribute-to-global-nature-goals-and-targets/' target='blank'>
+            CitiesWithNature Action Platform
+            </a> 
```

Comment:
This line break makes the space after the text part of the anchor tag

Reply:
fixed